src/common/datavcmn.cpp

```diff
@@ -334,8 +334,8 @@ int wxDataViewModel::Compare( const wxDataViewItem &item1, const wxDataViewItem
                               unsigned int column, bool ascending ) const
 {
     wxVariant value1,value2;
-    GetValue( value1, item1, column );
-    GetValue( value2, item2, column );
+    HasValue(item1, column) ? GetValue( value1, item1, column ) : value1.MakeNull();
+    HasValue(item2, column) ? GetValue( value2, item2, column ) : value2.MakeNull();
```

Comment:
I don't see why do we need to call `MakeNull()` here, this seems to be redundant and using ternary operator without using its return value is rather strange, so I'm changing it to just use a simple `if`.

Reply:
Thank you. Good catch.